### DISCUSSION TITLE
Help with people sizing prompt

### DISCUSSION THREAD

**Discussion by PlokAi1** [lllyasviel/Fooocus#1296] [General]
Hi,
I'm trying to create an image of 2 kids brother and sister when sister is older and bigger, no matter what i try the boy will always look bigger, like the AI finds it illogical for the girl to be bigger or taller for that matter.
Any workaround?

**Comment by Hypersonic-Popcorn** [comment]
I think in Fooocus itself, about all you can do is use inpainting to make sure that the boy is the smaller one. Getting details right is always hard with SD.

**Comment by PlokAi1** [comment]
Well ive tried that but even in in-painting when i ask the make the girl bigger it does not work or sometimes it just make it a boy
